### DISCUSSION TITLE
FOXESS USB LAN Dongle / Adaptor

### DISCUSSION THREAD

**Discussion by DerRonnyUK** [nathanmarlor/foxess_modbus#632] [General]
Hi Folks,

what do I need to do to connect the Foxess Lan Dongle?

**Comment by canton7** [comment]
You don't I'm afraid, the dongles only connect to the foxess cloud. You need to connect to the modbus interface using one of the methods described in the documentation.

**Reply by DerRonnyUK** [reply]
thank you very much for your swift Reply - an RS485 to ethernet adaptor it is then ;-) 
likely have to go then for USR-TCP232-410s - as I have a LAN connection to a router nearby anyway - no need (and not a fan of ) Wifi connection.
Thanks Again

**Reply by FozzieUK** [reply]
Good choice, or go with the Waveshare if you have PoE and don't have a power socket near the inverter.

**Reply by canton7** [reply]
(That said, PoE adapters are cheap. I use one with a USR-W610)

**Reply by DerRonnyUK** [reply]
rigt ! one step further - the old Router will have to say ciao - and a new gigabit Router with PoE from TP Link will replace the D-Link
however - one question. In the Guide for the Wiring - at least for me it`s missing. Is it really that simple to connect a cable to the 485 Pins and then to the waveshare rs485 to ETH adaptor?  on the A and B?

**Reply by FozzieUK** [reply]
Yep it’s really that simple - it’s just a 2 wire balanced connection and doesn’t need power or ground, for short distances it will work on twin core cables, but twisted pair is better.

**Reply by DerRonnyUK** [reply]
working now - can be closed from my end
